distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -550,26 +551,130 @@ type Column
                 op_result = self.make_op "IIF" [when_true, when_false] new_name
                 adapt_unified_column op_result common_type
 
-    ## Rounding values is not supported in database columns.
-    round : Integer -> Boolean -> Problem_Behavior -> Column | Illegal_Argument | Invalid_Value_Type
-    round self decimal_places=0 use_bankers=False on_problems=Report_Warning =
-        _ = [decimal_places, use_bankers, on_problems]
-        Error.throw <| Unsupported_Database_Operation.Error "`Column.round` is not implemented yet for the Database backends."
+    ## Creates a column from a single constant value. The value must be numeric,
```

Comment:
For now, I'd make this `PRIVATE` until we figure out what the API should be. I'm not sure if it should be a non-static method if it is supposed to be public (but for internal it can be, I know making it static would complicate some logic)

Reply:
Done.